src/styles/utills/variables.scss

```diff
@@ -0,0 +1,4 @@
+$white: #fff;
+$blue: #00acdc;
+$black: #060b35;
+$grey: #616070;
```

Comment:
it is wrong variable names,  if I decided to change the blue button to pink, I would have to change it everywhere, not just in the variable definition. You need to fix the variable names, they should show their purpose, not their content.